### ISSUE TITLE
Reproduce MoCov2 on ImageNet 1k

### ISSUE THREAD

**Issue by lianqing01** [HobbitLong/CMC#49]
Hi @HobbitLong , I am trying to reproduce MoCo v2 on ImageNet 1k. Have you tried to replace the Linear projection head to MLP? Do you think it is necessary that add the batch normalization layer or bias for the fully connected layer? I keep all the hyper-param same as the paper but only could get 61.4~ acc with 4 gpu 256 batch size. 

Would you kindly share with me the specific configurations based on your codebase for reproducing MoCov2-ResNet-50?

Thanks a lot!

**Comment by HobbitLong** [comment]
@lianqing01 , I guess you already figure it out.

But in case you are interested, I have open-sourced another repo contains many current SoTA methods (including MoCo v1, v2, etc), and will contain some more. Please see [here](https://github.com/HobbitLong/PyContrast)
